<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>password_change</class>
 <widget class="QDialog" name="password_change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	font-family: 'Gothic A1', sans-serif;
}

QLabel#title_label {
	text-align: center;
	font-weight: bold;
	font-size: 18px;
}

QDialog#auth_window {
	background-color: #252526;
}

QLabel {
	color: #f1f1ef;
}

QLineEdit {
	background-color: #333337;
	color: #f1f1ef;
}

QLabel#login_label {
	font-size: 14px;
}

QLabel#password_label {
	font-size: 14px;
}

QPushButton#auth_button {
	border: 2px solid gray;
	border-radius: 10px;
	background-color: #252526;
	color: #f1f1ef;
	padding: 3px;
}</string>
   </property>
   <property name="text">
    <string>Измененение пароля</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="old_pas_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="new_pas">
      <property name="text">
       <string>Новый пароль</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="old_pas">
      <property name="text">
       <string>Старый пароль</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="new_pas_edit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="accept_pas_edit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="accept_pas">
      <property name="text">
       <string>Подтвердить пароль</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="acc">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rej">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
